sites/upsun/src/add-services/postgresql.md

```diff
@@ -757,6 +757,13 @@ For more details, see how to [upgrade to PostgreSQL 12 with `postgis`](#upgrade-
 
 {{< /note >}}
 
+{{< note >}}
```

Comment:
I worry, given the length of the extensions section, that adding the note to the bottom will be completely missed by readers. When they get to the `adminpack` extension in the list, the note is still not in the viewport.